MANIFEST.in

```diff
@@ -1,4 +1,5 @@
 include dash_bio/bundle.js
 include dash_bio/metadata.json
 include dash_bio/package.json
+include dash_bio/genomeviewer-style.css
```

Comment:
It's kind of odd to have this here, since none of the other components have stylesheets. Is there a workaround at all? (Also, is this stylesheet for the component or the demo app?)

Reply:
There is no `dash_bio/genomeviewer-style.css` file, so I'll just remove this line!